asyncapi/hubspot/Module.md

```diff
@@ -0,0 +1,59 @@
+## Overview
+
+The [Ballerina](https://ballerina.io/) listener for Hubspot provides capability to listen following events happening in HubSpot account.
+* `onCompanyCreation`
+* `onCompanyDeletion`
+* `onContactCreation`
+* `onContactDeletion`
+* `onConversationCreation`
+* `onConversationDeletion`
+* `onDealCreation`
+* `onDealDeletion`
+
+## Prerequisites
+
+Before using this connector in your Ballerina application, complete the following:
+
+- Create [Hubspot account](https://app.hubspot.com/signup-hubspot/)
+- Create an [app](https://developers.hubspot.com/get-started) in devoloper portal
+- Go to webhooks and register a targetURL for notifications.
+- Create a subscription with interested events.
+
+## Quickstart
+To use the Hubspot listener in your Ballerina application, update the .bal file as follows:
+
+### Step 1: Import listener
+Import the ballerinax/trigger.hubspot module into the Ballerina project.
+```ballerina
+    import ballerinax/trigger.hubspot;
+```
+
+### Step 2: Create a new listener instance
+Create a `listenerConfig` with the AppSecret and CallbackURL. 
+```ballerina
+    listener hubspot:Listener hubspotWebhook = new(listenerConfig = {clientSecret, callbackURL});
+```
+**Note:** `ClientSecret` can be obtained from the Hubspot portal.
+
+### Step 3: Invoke listener triggers
+1. Now you can use the triggers available within the listener. 
+
+    Following is an example of how to listen for events on Company creation and deletion.
+
+    Listen to Hubspot Company Creation and Deletion
```

Comment:
Is this a comment within the `.bal` file? If not, shall we leave it out because it causes a repetition?